### PR TITLE
Add LlamaHub load component

### PR DESCRIPTION
**Pull request by RobbeSneyders**
This PR adds a load component that can load data using any LlamaHub loader.

Opening as a draft PR for now since it uses the dev release.

### CODE REVIEW

**Comment by RobbeSneyders** [comment]
Ready to be reviewed and merge, since 0.8.dev2 is in a state that we can start merging updated PRs here.

File: src/components/load_with_llamahub/src/main.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        for requirement in additional_requirements:
+            subprocess.check_call(  # nosec
+                [sys.executable, "-m", "pip", "install", requirement],
+            )
+
+    def get_columns_to_keep(self) -> t.List[str]:
+        # Only read required columns
+        columns = []
+
+        if self.column_name_mapping:
```
Comment (Hakimovich99) [inline src/components/load_with_llamahub/src/main.py:61]: But self.column_name_mapping is never defined?

Comment (Hakimovich99) [inline src/components/load_with_llamahub/src/main.py:61]: But self.column_name_mapping is never defined?

**Review by Hakimovich99 — CHANGES_REQUESTED** [review]

File: src/components/load_with_llamahub/src/main.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+            invert_column_name_mapping = {
+                v: k for k, v in self.column_name_mapping.items()
+            }
+        else:
+            invert_column_name_mapping = {}
+
+        for field_name, field in self.spec.produces.items():
```
[the remark this answers is not in the document]
Comment (RobbeSneyders) [reply src/components/load_with_llamahub/src/main.py]: Also removed.

**Comment by RobbeSneyders** [comment]
Both your comments should be addressed @Hakimovich99. Retested the component, still works :+1:

**Review by Hakimovich99 — APPROVED** [review]

**Pull request merged**